src/rules/no_redeclare.rs

```diff
@@ -40,6 +40,49 @@ impl LintRule for NoRedeclare {
       ProgramRef::Script(ref s) => visitor.visit_script(s, &DUMMY_NODE),
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows redeclaration of variables.
```

Comment:
This rule seems to affect also parameter, function, and class property declarations. Perhaps this sentence should be changed?

Reply:
Good point, I missed those! I'll fix it today.

Reply:
@dsherret Done, please take a look again.